DCC.Core.Types.schema.json

```diff
@@ -29,39 +22,39 @@
       "type": "object",
       "properties": {
         "fn": {
-          "title": "Family name",
-          "description": "The family or primary name(s) of the person addressed in the certificate",
+          "title": "Surname",
+          "description": "The surname or primary name(s) of the person addressed in the certificate",
           "type": "string",
-          "maxLength": 50,
+          "maxLength": 80,
           "examples": [
             "d'Červenková Panklová"
           ]
         },
         "fnt": {
-          "title": "Standardised family name",
-          "description": "The family name(s) of the person transliterated",
+          "title": "Standardised surname",
+          "description": "The surname(s) of the person, transliterated ICAO 9303",
```

Comment:
A comment on how to truncate? Perhaps not in the description, but somewhere in the wiki? ICAO 9303, part 4.

Reply:
Is already given in the approved documentation:

> Surname(s) of the holder transliterated using the same
> convention as the one used in the holder’s machine
> readable travel documents (such as the rules defined in
> ICAO Doc 9303 Part 3).

... but then who reads the docs anyway? Come to think of it, who reads FAQs either... ;)

Still if you think it will be generally helpful, then I'll add sth to the wiki/FAQ

Reply:
https://github.com/ehn-dcc-development/ehn-dcc-schema/wiki/FAQ#name-transliteration-and-truncation